src/main/java/com/codeborne/selenide/ClipboardService.java

```diff
@@ -0,0 +1,38 @@
+package com.codeborne.selenide;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.ServiceLoader;
+import java.util.stream.StreamSupport;
+
+import static com.codeborne.selenide.impl.Plugins.getDefaultPlugin;
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+
+public class ClipboardService {
+
+  private Driver driver;
+
+  public ClipboardService(Driver driver) {
+    this.driver = driver;
+  }
+
+  public Clipboard load() {
+    if (isNullOrEmpty(driver.config().remote())) {
+      return initClipboardWithDriver(getDefaultPlugin(Clipboard.class).getClass());
+    } else {
+      Clipboard implementation = StreamSupport.stream(ServiceLoader.load(Clipboard.class).spliterator(), false)
+        .findFirst()
+        .orElseThrow(() -> new IllegalStateException("Remote clipboard plugin not defined!"));
+      return initClipboardWithDriver(implementation.getClass());
+    }
+  }
```

Comment:
this code can be simplified. 
1. let's just have class `ClipboardService` with default constructor and method `public Clipboard getClipboard(Driver driver)`.
2. project `selenide-selenoid` will have a subclass `SelenoidClipboardService` with overridden method `getClipboard`. 
3. whoever needs `ClipboardService` will just call `inject(ClipboardService.class)` to get it.

Reply:
@asolntsev 
Do you mean `Plugins.inject()`? 
In my case i didn't use it because my logic was that user often has two types of running test:
1) Local - when writing or debug
2) Remote - running on CI etc.

And i don't want to change configuration (enable and disable plugin) every time when run tests in different modes. 
`Plugins.inject()` works as if there are no services in `META-INF.services` loads default from, but i want to load default during local run and use custom during remote run. So `Plugins.inject()` isn't suitable for this logic )
Are you agree, or maybe i'm not clearly understood you?)

Reply:
Done as was requested:
- made `ClipboardService` with default constructor and method getClipboard(Driver driver)